llvm/lib/Target/ARM/MCTargetDesc/ARMELFStreamer.cpp

```diff
@@ -670,17 +670,15 @@ class ARMELFStreamer : public MCELFStreamer {
   }
 
   void EmitMappingSymbol(StringRef Name) {
-    auto *Symbol = cast<MCSymbolELF>(getContext().getOrCreateSymbol(
-        Name + "." + Twine(MappingSymbolCounter++)));
+    auto *Symbol = cast<MCSymbolELF>(getContext().createLocalSymbol(Name));
```

Comment:
I think these two functions are the only use of MappingSymbolCounter (similar to AArch64). Can it be removed?

Reply:
Removed!